### PR TITLE
data:luckyklyist.json

### PR DESCRIPTION
**Pull request by luckyklyist**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6107"></a>

### CODE REVIEW

File: data/luckyklyist.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/luckyklyist"
+		},
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/luckyklyist"
+		},
+		{
+			"icon": "FaYoutube",
+			"url": "https://youtube.com/codewithklyist"
```
Comment (imbeshat) [inline data/luckyklyist.json:22]:
```suggestion
			"url": "https://youtube.com/@codewithklyist"
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Review by shyamtawli — COMMENTED** [review]
LGTM.
But you can remove the extra spaces. So that it will look clean.

File: data/luckyklyist.json (hunk 1 above)
Comment (shyamtawli) [reply data/luckyklyist.json:22]: nice catch, @imbeshat

**Review by md-abid-hussain — APPROVED** [review]
LGTM

**Review by harshsingh32 — APPROVED** [review]

File: data/luckyklyist.json (hunk 1 above)
Comment (kumarsonsoff3) [reply data/luckyklyist.json:22]: Hey @luckyklyist, do you need any help in making the above requested changes!?

**Review by harshsingh32 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.eddiehub.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**